mmv1/templates/terraform/examples/secure_source_manager_repository_initial_config.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_secure_source_manager_instance" "instance" {
+    location = "us-central1"
+    instance_id = "<%= ctx[:vars]['instance_id'] %>"
+}
+
+resource "google_secure_source_manager_repository" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Per the missing test message, it might be worth making this a "full" example, and include every possible field (like `etag`). Not strictly necessary if it doesn't make sense, but we do strongly prefer to have all configurable fields be used in at least one test.

Reply:
Our API response currently doesn't have an etag field. So I think we can leave it for now unused. For the other missing test, we also don't support IAM conditions either.

Reply:
Yea, the iam conditions are fine to ignore.

For the `etag` though, it is a bit strange to be adding it here if it can't be used. I would definitely consider removing it from the PR, unless there are plans to add it in the near future. It can always be added later (in practice, it might not be a very useful field anyway).

Reply:
Good points, I have removed that field. 👍